src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -49,6 +49,8 @@ public class EditCommand extends Command {
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_FIELD = "Detected multiple modification to a same parameter, "
+            + "applying the latest change.\n";
```

Comment:
modification -> modifications?
Is it correct to say _modifications_ i.e., are these strictly modifications or something else? Similarly, 'latest change' doesn't seem like the correct phrase.